fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImpl.java

```diff
@@ -57,8 +58,9 @@
      * @throws IOException If we can't access the file.
      */
     public FedoraToOCFLObjectIndexImpl() throws IOException {
-        if (FEDORA_TO_OCFL_INDEX_FILE.exists() && FEDORA_TO_OCFL_INDEX_FILE.canRead()) {
-            try (var lines = Files.lines(FEDORA_TO_OCFL_INDEX_FILE.toPath())) {
+        fedoraToOcflIndexFile = new OCFLConstants().getFedoraToOCFLIndexFile();
```

Comment:
I have given up on avoiding all these instantiations. There is no way to `@Inject` into this class in a test without changing from a Mockito runner to using a Spring test runner and giving `@ContextConfiguration` to define all the bits because this constructor needs the underlying class right as the class is created.